everrest-websockets/src/main/java/org/everrest/websockets/message/MessageSender.java

```diff
@@ -140,6 +146,11 @@ String getText() {
     private class MessageSendHandler implements SendHandler {
         @Override
         public void onResult(SendResult result) {
+            LOG.debug(" SendQueue size {} ,  maxNumberOfMessageInQueue {} result {}",
```

Comment:
This will be called on each message sending. It is not efficient IMO even in non-debug logging mode.